bin/compare_threshold.py

```diff
@@ -149,39 +122,44 @@ def main(args=None):
 
     # Required mean depth of coverage to pass the validation
     mean_depth_of_converage_required = float(thresholds["meanDepthOfCoverage"])
-
     # Read mosdepth summary file
     df = pd.read_csv(args.mosdepth_global_summary, sep="\t")
-    row_name = "total_region" if args.libraryType in ["panel", "wes","panel_lr", "wes_lr"] else "total"
+    row_name = "total_region" if args.libraryType in ["panel", "wes", "panel_lr", "wes_lr"] else "total"
     mean_depth_of_coverage = df.loc[df["chrom"] == row_name, "mean"].item()
 
     # --- Determine 'percentBasesAboveQualityThreshold' ---
 
     # Base quality threshold
-    quality_threshold = thresholds["percentBasesAboveQualityThreshold"][
-        "qualityThreshold"
-    ]
-    # Minimum percentage of bases above the quality threshold required to pass the validation
+    quality_threshold = thresholds["percentBasesAboveQualityThreshold"]["qualityThreshold"]
     percent_bases_above_quality_threshold_required = thresholds[
-        "percentBasesAboveQualityThreshold"
-    ]["percentBasesAbove"]
+        "percentBasesAboveQualityThreshold"]['percentBasesAbove']
 
-    # read fastp json file
-    with open(args.fastp_json, "r") as f:
-        fastp_data = json.load(f)
-    fastp_filtering_stats = fastp_data["summary"]["before_filtering"]
+    total_bases = 0
+    total_bases_above_quality = 0
 
-    if f"q{quality_threshold}_rate" not in fastp_filtering_stats:
-        raise ValueError(
-            f"'q{quality_threshold}_rate' not found in fastp summary!\n"
-            f"-> Could not determine percentBasesAboveQualityThreshold for 'qualityThreshold': {quality_threshold}."
-        )
-    fraction_bases_above_quality_threshold = fastp_filtering_stats[
-        f"q{quality_threshold}_rate"
-    ]
-    percent_bases_above_quality_threshold = fraction_bases_above_quality_threshold * 100
+    # read fastp json files
+    for fastp_json_file in args.fastp_json:
+        with open(fastp_json_file, "r") as f:
+            fastp_data = json.load(f)
+        fastp_filtering_stats = fastp_data["summary"]["before_filtering"]
+
+        if f"q{quality_threshold}_rate" not in fastp_filtering_stats:
+            raise ValueError(
+                f"'q{quality_threshold}_rate' not found in fastp summary: {fastp_json_file}\n"
+                f"-> Could not determine percentBasesAboveQualityThreshold for 'qualityThreshold': {quality_threshold}."
+            )
+
+        file_total_bases = fastp_filtering_stats["total_bases"]
+        file_q_rate = fastp_filtering_stats[f"q{quality_threshold}_rate"]
 
-    # --- Determine 'targetedRegionsAboveMinCoverage' ---
+        total_bases += file_total_bases
+        total_bases_above_quality += file_total_bases * file_q_rate
```

Comment:
```suggestion
        total_bases_above_quality += fastp_filtering_stats[f"q{quality_threshold}_bases"]
```

Can use the total bases above quality reported by fastp directly instead of approximating by multiplying.